awx/main/models/notifications.py

```diff
@@ -185,6 +185,11 @@ class Meta:
         default='',
         editable=False,
     )
+    timeout = models.IntegerField(
```

Comment:
Don't think we need this anymore, see big comment above.

Reply:
Removed and tested, everything was fine so this requested change has been made!